src/web/components/Contributors/Contributors.tsx

```diff
@@ -0,0 +1,66 @@
+import { Card, Button, Text, Avatar } from "@fluentui/react-components";
+import "./Contributors.css";
+
+function ContributorsSection() {
+    const contributors = [
```

Comment:
How is this generated? We have a script that updates these randomly thru the week. I would rather not have a copy of them if we can help it. I can configure the tool to output HTML in our TSX file. Would that be okay? I'm assuming we can use CSS to style it, right? Worst case, we can add a build script that copies this from the .allcontributors file where it lives.

Reply:
We'll fix this later